<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YahooSearchBase</class>
 <widget class="QWidget" name="YahooSearchBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="grpID" native="true">
     <property name="title" stdset="0">
      <string>Yahoo! &amp;ID</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLineEdit" name="edtID"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="grpName" native="true">
     <property name="title" stdset="0">
      <string>&amp;Name</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLineEdit" name="edtName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="grpKeyword" native="true">
     <property name="title" stdset="0">
      <string>&amp;Keyword</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLineEdit" name="edtKeyword"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox5">
     <property name="title">
      <string>Conditions</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="lblGender">
        <property name="text">
         <string>Gender:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbGender"/>
      </item>
      <item>
       <widget class="QLabel" name="lblAge">
        <property name="text">
         <string>Age:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbAge"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="Spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
